src/components/organisms/WithNavigationBar/WithNavigationBar.tsx

```diff
@@ -84,19 +71,14 @@ export const WithNavigationBar: React.FunctionComponent<PropsType> = ({
           </span>
         </OverlayTrigger>
       )}
-
-      <NavbarRadio />
```

Comment:
As called out in another comment in this review, `NavbarRadio` should also be injected here, only for the 'user-side' of things (eg. not admin)